main/commons/src/test/java/org/cryptomator/common/settings/VaultSettingsTest.java

```diff
@@ -15,12 +15,19 @@
 public class VaultSettingsTest {
 
 	@Test
-	public void testNormalize() throws Exception {
-		assertEquals("_", VaultSettings.normalizeMountName(" "));
-		assertEquals("a", VaultSettings.normalizeMountName("ä"));
-		assertEquals("C", VaultSettings.normalizeMountName("Ĉ"));
-		assertEquals("_", VaultSettings.normalizeMountName(":"));
-		assertEquals("_", VaultSettings.normalizeMountName("汉语"));
+	public void testNormalize() {
+		VaultSettings settings = new VaultSettings("id");
+		settings.displayName().setValue(" ");
+		assertEquals("_", settings.normalizeDisplayName());
+
+		settings.displayName().setValue("ä");
+		assertEquals("a", settings.normalizeDisplayName());
+		settings.displayName().setValue("Ĉ");
+		assertEquals("C", settings.normalizeDisplayName());
+		settings.displayName().setValue(":");
+		assertEquals("_", settings.normalizeDisplayName());
+		settings.displayName().setValue("汉语");
+		assertEquals("_", settings.normalizeDisplayName());
```

Comment:
Make it a ParameterizedTest

Reply:
Implemented with https://github.com/cryptomator/cryptomator/pull/1318/commits/eb6ade49f680e0767816163fe74df0dacda3564c